.github/workflows/skore.yml

```diff
@@ -1,10 +1,14 @@
-name: Reusable backend workflow
+name: Reusable skore workflow
```

Comment:
I'm not found of the naming, especially has it appears in the badges. 
May I suggest to simply call the actions skore and UI ?

Reply:
These workflows will not longer be displayed as badges https://github.com/probabl-ai/skore/pull/444

Reply:
But if you want I can rename them.